experiments/modern/src/initialize.js

```diff
@@ -1,8 +1,31 @@
 import * as Utils from "./utils";
-import JSZip from "jszip";
+
+function splitValues(s) {
+  const retArr = [];
+  const arr = s.split(",");
+  for (let i = 0; i < arr.length; i++) {
+    retArr.push(arr[i]);
+  }
+  return arr.map((v) => parseFloat(v));
+}
```

Comment:
`retArr` is never used?

```
function splitValues(str) {
    return str.split(",").map(parseFloat);
}
```

Reply:
lol, nice catch. yay extra for loop hah!